api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -582,6 +588,35 @@ message HttpConnectionManager {
   // *not* the deprecated but similarly named :ref:`stream_error_on_invalid_http_messaging
   // <envoy_v3_api_field_config.core.v3.Http2ProtocolOptions.stream_error_on_invalid_http_messaging>`
   google.protobuf.BoolValue stream_error_on_invalid_http_message = 40;
+
+  // The configuration for the original-IP detection extensions.
+  //
+  // By default, when the :ref:`use_remote_address
+  // <envoy_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.use_remote_address>`
```

Comment:
should we update the defaults now the recommend way to configure xff_num_trusted_hops is changed? 

Could be the default is using deprecated legacy blah but once blah is removed, the default will be [behavior of default]?

Maybe point to example config with the "default" extension and xff_num set?

Reply:
Ah yes, to 90% of this :-) 

Given that, the configuration changes but the behavior remains.